.github/workflows/python-eks-e2e-test.yml

```diff
@@ -174,6 +191,10 @@ jobs:
               -var="service_account_aws_access=service-account-${{ env.TESTING_ID }}" \
               -var="python_app_image=${{ env.MAIN_SAMPLE_APP_IMAGE_ARN }}" \
               -var="python_remote_app_image=${{ env.REMOTE_SAMPLE_APP_IMAGE_ARN }}" \
+              -var="rds_mysql_cluster_endpoint=${{env.RDS_MYSQL_CLUSTER_ENDPOINT}}" \
+              -var="rds_mysql_cluster_username=${{env.RDS_MYSQL_CLUSTER_SECRETS_USERNAME}}" \
+              -var="rds_mysql_cluster_password=${{env.RDS_MYSQL_CLUSTER_SECRETS_PASSWORD}}" \
+              -var="rds_mysql_cluster_database=information_schema" \
```

Comment:
Just confirming, this won't cause a failure if we don't deploy the changes to the sample app right? From my understanding it will just add environment variables that won't be used if we don't update the sample app?

Reply:
That's right, the sample app won't fail if we don't update the version in ECR straightaway. What could happen is that in the GitHub workflow step that performs the curl, will return Error 404 page not found when trying to call the mysql endpoint, although it won't fail the test run anyway because of [this configuration](https://github.com/aws-observability/aws-application-signals-test-framework/blob/main/.github/workflows/python-eks-e2e-test.yml#L309).